lms/templates/dashboard/_dashboard_status_verification.html

```diff
@@ -38,7 +38,7 @@
 
 %endif
 
-%if verification_status == 'must_reverify':
+%if verification_status == 'must_reverify' or verification_status == 'expired':
```

Comment:
You could write this more concisely as `if verification_status in ['must_reverify', 'expired']`.